### ISSUE TITLE
[Feature Request] print_tensors_in_checkpoint_file should accept Google Cloud Bucket addresses of the form 'gs://...'

### ISSUE THREAD

**Issue by vedal** [tensorflow/tensorflow#19616]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
MacOS High Sierra
- **TensorFlow installed from (source or binary)**:
binary
- **TensorFlow version (use command below)**:
v1.8.0-1660-ga543d94710;  1.9.0-dev20180510
- **Python version**:
3.6.4 
Bazel version:
N/A
CUDA/cuDNN version:
N/A
GPU model and memory:
N/A
Exact command to reproduce:
```python
from tensorflow.python.tools import inspect_checkpoint as chkp
chkp.print_tensors_in_checkpoint_file("gs://...",all_tensors=True)
```

### Describe the problem
The function `print_tensors_in_checkpoint_file` does not seem to handle Google Cloud Bucket addresses of the form 'gs://...', but rather gives a "not found" error. This does not match the behavior of tf.train.Saver.save() and .restore() 

### Source code / logs
N/A

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 16 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by amundv** [comment]
I edited the first comment as requested

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 59 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 74 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 89 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 104 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 119 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 134 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 149 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @robieta: It has been 167 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tilakrayal** [comment]
@vedal ,
Is this still an issue?
Could you update TensorFlow to the latest stable version v2.9 and let us know if you are facing the same error. Thank you!

**Comment by vedal** [comment]
Thanks @tilakrayal for picking up this thread. I unfortunately don't have access to the GCS resources I used at the time, so I cannot test this on my side.

**Comment by tilakrayal** [comment]
@vedal,
Could you please try to test and let us know if the issue still persists? Thank you!

**Comment by vedal** [comment]
> Thanks @tilakrayal for picking up this thread. I unfortunately don't have access to the GCS resources I used at the time, so I cannot test this on my side.

@tilakrayal there's not more I can do here. Please feel free to close the issue :)

**Comment by tilakrayal** [comment]
@vedal,
As [commented](https://github.com/tensorflow/tensorflow/issues/19616#issuecomment-1257183785) by the user, we are moving this issue to closed status. Thank you!

**Comment by tilakrayal** [comment]
@vedal,
As [commented](https://github.com/tensorflow/tensorflow/issues/19616#issuecomment-1257183785) by the user, we are moving this issue to closed status. Thank you!
